schemas/2018-09-01/Microsoft.Insights.json

```diff
@@ -0,0 +1,460 @@
+        }
+      },
+      "required": [
+        "name",
+        "workspaceId",
+        "connectionId",
+        "ticketConfiguration",
+        "region"
+      ],
+      "description": "An Itsm receiver."
+    },
+    "AzureAppPushReceiver": {
+      "type": "object",
+      "properties": {
+        "name": {
+            "type": "string",
+            "description": "The name of the Azure mobile app push receiver. Names must be unique across all receivers within an action group."
+        },
+        "emailAddress": {
+            "type": "string",
+            "description": "The email address registered for the Azure mobile app."
+        }
+      },
+      "required": [
+        "name",
+        "emailAddress"
+      ],
+      "description": "The Azure mobile App push notification receiver."
+    },
+    "AutomationRunbookReceiver": {
+      "type": "object",
+      "properties": {
+      "automationAccountId": {
+              "type": "string",
+              "description": "The Azure automation account Id which holds this runbook and authenticate to Azure resource."
+          },
+        "runbookName": {
+            "type": "string",
+            "description": "The name for this runbook."
+        },
+        "webhookResourceId": {
+            "type": "string",
+            "description": "The resource id for webhook linked to this runbook."
+        },
+        "isGlobalRunbook": {
+            "type": "boolean",
+            "description": "Indicates whether this instance is global runbook."
+        },
+        "name": {
+            "type": "string",
+            "description": "Indicates name of the webhook."
+        },
+        "serviceUri": {
+            "type": "string",
+            "description": "The URI where webhooks should be sent."
+        }
+      },
+      "required": [
+        "automationAccountId",
+        "runbookName",
+        "webhookResourceId",
+        "isGlobalRunbook"
+      ],
+      "description": "The Azure Automation Runbook notification receiver."
+    },
+    "VoiceReceiver": {
+      "type": "object",
+      "properties": {
+			"name": {
+                "type": "string",
+                "description": "The name of the voice receiver. Names must be unique across all receivers within an action group."
+            },
+            "countryCode": {
+                "type": "string",
+                "description": "The country code of the voice receiver."
+            },
+            "phoneNumber": {
+                "type": "string",
+                "description": "The phone number of the voice receiver."
+            }
+      },
+      "required": [
+        "name",
+        "countryCode",
+        "phoneNumber"
+      ],
+      "description": "A voice receiver."
+    },
+    "LogicAppReceiver": {
+      "type": "object",
+      "properties": {
+			"name": {
```

Comment:
>			 [](start = 0, length = 3)

use spaces.

Reply:
please format the json file. 

---
In reply to: [215785063](https://github.com/Azure/azure-resource-manager-schemas/pull/502#discussion_r215785063) [](ancestors = 215785063)